Materie/SO/README.md

```diff
@@ -9,6 +9,10 @@
 - [Appunti di teoria di Paolo Alfano](Alfano_2015-2016_Teoria.pdf) (AA 2015-2016)
 - [Appunti di laboratorio di Paolo Alfano](Alfano_2015-2016_Laboratorio.pdf) (AA 2015-2016)
 
+## ESERCIZI DI LABORATORIO
+
+- [Esercizi di laboratorio di Mirco De Andreis](https://github.com/MircoDeAndreis/ESERCIZI-LABORATORIO-SISTEMI-OPERATIVI-TURNO1-2024-2025/tree/2b256977af42ee7cb264289481bc142a652ff3b3) (AA 2024-2025)
```

Comment:
In questo caso credo che il link diretto senza includere lo specifico tree del commit vada bene